### ISSUE TITLE
Windows compatibility

### ISSUE THREAD

**Issue by zerone01y** [bordaigorl/remy#36]
Hi, do you consider making it compatible with Windows? Thanks.

**Comment by bordaigorl** [comment]
It should already be compatible with it 🤔 
Can you tell me what goes wrong if you try to install it?

**Comment by zerone01y** [comment]
Hi,

When I ran it on Win10, the logs say:

```
[I] Connecting...
[I] Connected to 172.26.59.219
[E] RETRY? [[Errno 2] No such file]
```

Seems it connects to the device successfully before throwing out the weird error. I had some success on my mbp so I guess it's a compatibility issue.

**Comment by JustLikeSprite** [comment]
Hi, I encounter the same problem just like you. I want to know what you do before meeting the "Error 2". Here is my way:

I tried to generate a pair of SSH key and uploaded the public key to the ~/.ssh/authorized_keys on remarkable. Afterwards I set the ssh source as the following:

> 
    "sources": {
        "ssh":
        {
            "name": "reMarkable (WiFi)",
            "type": "ssh",
            "host": "10.11.99.1",   
            "key": "C:/Users/.ssh/reMarkable_id_rsa",
            "username": "root",
            "timeout": 3,
            "persist_cache": true,
            "use_banner": "remy-banner.png"
          }
   "C:/Users/.ssh/reMarkable_id_rsa" is my private key. I thought it was done, but I failed and cannot figure out why.
Then I tried to change the option "key" to optional "password" and I assigned my root password to it. However, it still failed me.

**Comment by bordaigorl** [comment]
The problem was fixed by #37 which I just merged in master. If you pull the latest version from the repo it should work.

**Comment by JustLikeSprite** [comment]
> The problem was fixed by #37 which I just merged in master. If you pull the latest version from the repo it should work.

Yeah, you are a magician! I try to install again and I make it. Thank you for your effort to all the users of rM.
